frontend/src/pages/projects/screens/spawner/SpawnerPage.tsx

```diff
@@ -144,9 +144,11 @@ const SpawnerPage: React.FC<SpawnerPageProps> = ({ existingNotebook }) => {
               <NameDescriptionField
                 nameFieldId="workbench-name"
                 descriptionFieldId="workbench-description"
+                nameHelperText="Must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character"
```

Comment:
You may need to update the helper text here. It must start with an alphabetic character and
end with an alphanumeric character. Also, it accepts upper-case letters.